mhkit/wave/io/wind_toolkit.py

```diff
@@ -0,0 +1,251 @@
+import pandas as pd
+from rex import MultiYearWindX
+import matplotlib.pyplot as plt
+
+
+def region_selection(lat_lon, preferred_region=''):
+    '''
+    Returns the name of the predefined region in which the given coordinates reside.
+    Can be used to check if the passed lat/lon pair is within the WIND Toolkit hindcast dataset. 
+
+    Parameters
+    ----------
+    lat_lon : list or tuple
+        Latitude and longitude coordinates as floats or integers
```

Comment:
must be of length 2 correct? 

Seeing a type `list` it is not immediately clear that I cannot pass a list of tuples and this function would return multiple regions.

Reply:
Good point. I prefer to keep it as a single lat_lon pair, to avoid multiple sizes in ``preferred_region``. I'll remove list as an option so that this is clear